README.md

```diff
@@ -1,6 +1,6 @@
 ## GitHub Docs <!-- omit in toc -->
 
-This repository contains the documentation website code and Markdown source files for docs.github.com.
+This repository contains the documentation website code and Markdown source files for [docs.github.com](docs.github.com).
```

Comment:
Should use a proper URL:
```suggestion
This repository contains the documentation website code and Markdown source files for [docs.github.com](https://docs.github.com).
```

Alternatively also `<https://docs.github.com>`, which renders as: <https://docs.github.com>